### DISCUSSION TITLE
How to enable Prometheus receiver in Signoz?

### DISCUSSION THREAD

**Discussion by vedantyerpude** [SigNoz/signoz#4818] [General]
Hi Team, I am very interested in the query-service using PromQL to read data from Prometheus and Otel receiver in Signoz. I configured the otel-config.yaml for prometheus to target our source but its not working. 
 
Prometheus is on EC2 instance,
Springboot application is deployed on AWS AppRunner,
Signoz is self-hosted on EC2 instance
 
Can someone help with the setup of Prometheus and Signoz?

**Comment by nitishfy** [comment]
Hi @vedantyerpude , Did you get a chance to go through https://signoz.io/blog/opentelemetry-collector-prometheus-receiver/?
